app/scripts/move-popup/dimMoveItemProperties.directive.js

```diff
@@ -100,6 +101,12 @@
   function MoveItemPropertiesCtrl($sce, $q, storeService, itemService, settings, ngDialog, $scope, $rootScope, dimFeatureFlags, dimDefinitions) {
     var vm = this;
 
+    vm.getNumber = function(objective) {
```

Comment:
Remember, calling a function from within an Angular expression is super expensive. Could we precalculate this with a watch, or better, use a filter?